src/utilities/__tests__/extendSchema-test.js

```diff
@@ -359,6 +359,33 @@ describe('extendSchema', () => {
     expect(printExtensionNodes(someScalar)).to.deep.equal(extensionSDL);
   });
 
+  it('extends scalars by adding specifiedBy directive', () => {
+    const schema = buildSchema(`
+      type Query {
+        foo: Foo
+      }
+
+      scalar Foo
+
+      directive @foo on SCALAR
```

Comment:
Why do you need `@foo` here?
Since it's already tested in the above test case.

Reply:
@IvanGoncharov Good question.  The reason I added this was to validate the use-case when there were multiple `extend`s.

If we remove this directive, it would be possible for `src/utilities/extendSchema.js`'s `extendScalarType` method to have a naive implementation that only looked at the first extension and could miss this.  This is mostly testing the `extensions.map`/`extensions.find` part of `extendScalarType`.

How would you like to proceed?  I am happy to:
1. Simplify this test and remove this as suggested
2. Leave as is
3. Make this more robust by adding another directive to validate that the `extendScalarType` isn't simply looking at the last extension

Reply:
@m14t Thank you for an explanation, now it totally makes sense 👍